addons/mail/static/src/components/message/message.scss

```diff
@@ -373,6 +377,14 @@
     font-style: italic;
 }
 
+// Server will add <p> when processing the message
+// This will add the margin on temporary message to avoid flickers
+.o-pending {
+    .o_Message_prettyBody {
+        margin-bottom: $paragraph-margin-bottom;
+    }
+}
+
```

Comment:
You should have classname modifier `o-pending` on `o_Message_prettyBody`, so that you don't introduce CSS specificity.